app/assets/stylesheets/application.css.scss

```diff
@@ -91,16 +96,39 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
   }
 
   body {
+    background-color: $bg-color;
     margin: 0 0 20px;
   }
 
-}
+  form {
+    background: $white;
+    border: 1px solid $border;
+    border-collapse: inherit; 
+    border-radius: 5px;
+    margin-bottom:20px; 
+    padding: 20px;
+  }
+
+  #login-box form {
+    border:none;
+    border-radius: none;
+    margin: 0;
+    padding: 0;   
+  }
 
-// td.hover-actions {
-//   > a.action { display: none; }
-//   &:hover > a.action { display: inline; }
-// }
+  table.users tr:nth-of-type(2) th {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3